pulsar/transactionCoordinatorClient.go

```diff
@@ -0,0 +1,223 @@
+// Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
`pulsar/transactionCoordinatorClient.go` rename to `pulsar/transaction_coordinator_client.go`